pkg/stripe/certsfs.go

```diff
@@ -0,0 +1,8 @@
+//+build dev
+
+package stripe
+
+import "net/http"
+
+// FS exports the filesystem
+var CertsFS http.FileSystem = http.Dir("../../data/certs")
```

Comment:
Does this require that the certificate files are present locally to the CLI?

I was more expecting that we'd either embed the files via something like https://github.com/markbates/pkger or have them as embedded constant strings. Ideally the CLI is portable as a single golang binary, right?

Reply:
The cert files are embedded using https://github.com/shurcooL/vfsgen. The file below (`certsfs_vfsdata.go`) is the generated source file containing the actual data.

Reply:
Sorry, I missed that. Otherwise looks good